Lib/test/test_fractions.py

```diff
@@ -371,6 +380,40 @@ def testArithmetic(self):
         self.assertEqual(p.numerator, 4)
         self.assertEqual(p.denominator, 1)
 
+    def testLargeArithmetic(self):
```

Comment:
Oh, sorry, I was unclear. I wanted to see results of benchmarks for large numbers, not adding unittests for them. I think `testLargeArithmetic` is not needed.

Reply:
Do you think it's bad to have them? I don't think exercising the implementation with larger values is useless.

Reply:
It is up to you.